core/embed/lib/fonts/fonts.c

```diff
@@ -18,33 +18,36 @@
  */
 
 #include "fonts.h"
+#include <stdio.h>
```

Comment:
please don't include stdio - why is it needed?

Reply:
Deleted in [8ad8475](https://github.com/trezor/trezor-firmware/pull/3206/commits/8ad8475eea41f7b66cbcd2b95f969ed0677a65ec) - I used it so I can do `printf`